### ISSUE TITLE
⌨ `EmployeeSearchInput` button does not have discernible text

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#15046]
### ⌨ Accessibility and Expected Behaviour

`EmployeeSearchInput` button does not have discernible text.

### WCAG 2.1 Criteria

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html

### 📋 Steps to Reproduce

https://61e099a1bb1465003a73d3ce-uzsbepnsgk.chromatic.com/?path=/story/components-employeesearchinput--with-default-value

### 🕵️ Details

https://dequeuniversity.com/rules/axe/4.11/button-name?application=axeAPI

### ✅ Acceptance Criteria

- [ ] EmployeeSearchInput button has discernible text. "Clear search"

**Comment by petertgiles** [comment]
@esizer I just ran through the nomination flow and I didn't see any spots with more than one on the page.  Of course, we can't say it will never happen.  Would it make sense to use the default that Nienke provided and then allow a prop to change it on a case by case basis?  I'm guessing we would have the same problem with the "search work email" button.

**Comment by esizer** [comment]
> Would it make sense to use the default that Nienke provided and then allow a prop to change it on a case by case basis? I'm guessing we would have the same problem with the "search work email" button.

Well, I have a personally preferred method. Though, I'm not sure how it works for translation 😢 If it works, then it would be the best option. We essentially stitch two strings together. 

"Clear" + {label}. So for this instance it would be "Clear employee search". This works in English. Im just not sure how well it work sin French.

```html
<label id="employee-search">Employee search</label>
<input />
<button aria-labelledby="employee-search-clear employee-search">
  <span id="employee-search-clear" hidden>Clear</span>
</button>
```

**Comment by petertgiles** [comment]
I'm guessing that will be a problem due to masculine and feminine.  You won't know to use "Effacer la" or "Effacer le".  @NienkeBr any ideas?
